src/main/java/org/apache/datasketches/QuantilesDoublesAPI.java

```diff
@@ -21,121 +21,125 @@
    */
   double[] getCDF(double[] splitPoints, QuantileSearchCriteria searchCrit);
 
   /**
-   * Returns the maximum quantile of the stream. This is provided for convenience, but is distinct from the largest
-   * quantile retained by the sketch algorithm.
+   * Returns the maximum item of the stream. This is provided for convenience, but is distinct from the largest
+   * item retained by the sketch algorithm.
    *
    * <p>If the sketch is empty this returns NaN.</p>
    *
-   * @return the maximum quantile of the stream
+   * @return the maximum item of the stream
    */
-  double getMaxQuantile();
+  double getMaxItem();
 
   /**
-   * Returns the minimum quantile of the stream. This is provided for convenience, but is distinct from the smallest
-   * quantile retained by the sketch algorithm.
+   * Returns the minimum item of the stream. This is provided for convenience, but is distinct from the smallest
+   * item retained by the sketch algorithm.
    *
    * <p>If the sketch is empty this returns NaN.</p>
    *
-   * @return the minimum quantile of the stream
+   * @return the minimum item of the stream
    */
-  double getMinQuantile();
+  double getMinItem();
 
   /**
-   * Same as {@link #getPMF(double[], QuantileSearchCriteria) getPMF(splitPoints, INCLUSIVE)}
-   * @param splitPoints an array of <i>m</i> unique, monotonically increasing quantiles
-   * @return a PDF array of m+1 densities as doubles on the interval [0.0, 1.0).
+   * This is equivalent to {@link #getPMF(double[], QuantileSearchCriteria) getPMF(splitPoints, INCLUSIVE)}
+   * @param splitPoints an array of <i>m</i> unique, monotonically increasing items.
+   * @return a PMF array of m+1 probability masses as doubles on the interval [0.0, 1.0).
    */
   default double[] getPMF(double[] splitPoints) {
     return getPMF(splitPoints, INCLUSIVE);
   }
 
   /**
    * Returns an approximation to the Probability Mass Function (PMF) of the input stream
-   * as an array of densities as doubles on the interval [0.0, 1.0],
+   * as an array of probability masses as doubles on the interval [0.0, 1.0],
    * given a set of splitPoints.
-   * The sum of the densities in the returned array is always 1.0.
    *
    * <p>The resulting approximations have a probabilistic guarantee that can be obtained from the
    * getNormalizedRankError(true) function.
    *
    * <p>If the sketch is empty this returns null.</p>
    *
-   * @param splitPoints an array of <i>m</i> unique, monotonically increasing quantiles
-   * that divide the real number line into <i>m+1</i> consecutive non-overlapping intervals.
-   * It is not necessary to include either the minimum or maximum quantiles
-   * of the input stream in these split points.
+   * @param splitPoints an array of <i>m</i> unique, monotonically increasing items
+   * (of the same type as the input items)
+   * that divide the real number line into <i>m+1</i> consecutive, non-overlapping intervals.
```

Comment:
perhaps "input domain" instead of "real number line"

Reply:
Fixed.

Reply:
This is now fixed.